src/main/java/pacman/Pacman.java

```diff
@@ -1,2 +1,42 @@
-public class Pacman {
-}
\ No newline at end of file
+class Pacman extends Position {
+
+    //private int x, y;
+    public Pacman (int x, int y,boolean exist ){
+        super(x,y,exist);
+    }
+    
+    //functions those add and substract pacman's positions	
+    public void down(){
+        super.y++;
+    }
+    
+    public void right(){
+        super.x++;
+    }
+    
+    public void left(){
+        super.x--;
+    }
+    
+    public void up(){
+        super.y--;
+    }
+
+    public boolean isEatable(Dots dotsPosition){
+        if((super.getX() == dotsPosition.getX()) && (super.getY() == dotsPosition.getY()) && dotsPosition.exist){
```

Comment:
if possible, try to directly return condition result instead of using if-then-else